src/main/java/seedu/address/logic/commands/DeallocateMemberCommand.java

```diff
@@ -65,25 +65,29 @@ public CommandResult execute(Model model) throws CommandException {
             throw new CommandException(Messages.MESSAGE_INVALID_FACILITY_DISPLAYED_INDEX);
         }
         Member toBeDeallocated = lastShownMemberList.get(memberIndex.getZeroBased());
-        Facility toDeallocate = lastShownFacilityList.get(facilityIndex.getZeroBased());
+        Facility toDeallocateFrom = lastShownFacilityList.get(facilityIndex.getZeroBased());
 
-        if (!toDeallocate.isMemberAllocatedOnDay(toBeDeallocated, day)) {
+        handleDeallocation(toBeDeallocated, toDeallocateFrom, model);
+
+        String dayName = day.getDisplayName(TextStyle.FULL, Locale.getDefault());
```

Comment:
Recent merge just added DayUtil in commons so don't have to keep repeating `TestStyle.FULL, Locale.getDefault()`. Maybe you can overload the displayName there to use here  

```suggestion
        String dayName = displayDay(day);
```

Reply:
okay will do that thanks!